src/roman_datamodels/datamodels/_utils.py

```diff
@@ -206,8 +206,7 @@ def rdm_open(init, memmap=False, **kwargs):
             return init.copy(deepcopy=False)
 
         # Temp fix to catch JWST args before being passed to asdf open
-        if "asn_n_members" in kwargs:
-            del kwargs["asn_n_members"]
+        kwargs.pop("asn_n_members", None)
```

Comment:
@WilliamJamieson or @braingram - can you sign off on this change?

Reply:
Looks fine to me. It's from a new ruff rule: https://docs.astral.sh/ruff/rules/if-key-in-dict-del/